app/src/main/java/org/jellyfin/androidtv/integration/LeanbackChannelWorker.kt

```diff
@@ -32,14 +32,19 @@ import org.jellyfin.apiclient.model.querying.NextUpQuery
  *
  * More info: https://developer.android.com/training/tv/discovery/recommendations-channel
  */
-class ChannelManager(
+class LeanbackChannelWorker(
+	context: Context,
+	val workerParams: WorkerParameters,
 	private val apiClient: ApiClient
-) {
-	private companion object {
+) : CoroutineWorker(context, workerParams) {
+	companion object {
 		/**
 		 * Amount of ticks found in a millisecond, used for calculation
 		 */
 		private const val TICKS_IN_MILLISECOND = 10000
+
+		const val SINGLE_UPDATE_REQUEST_NAME = "channelUpdateRequest"
+		const val PERIODIC_UPDATE_REQUEST_NAME = "channelUpdateRequest"
```

Comment:
Why are these names the same?

Reply:
Since they are different request types it doesn't matter too much, I should've used different names for readability though.